src/ankh/cmd/ankh/main.go

```diff
@@ -234,11 +238,13 @@ func main() {
 		cmd.Command("templates", "For each chart, display contents of each raw template file",
 			func(cmd *cli.Cmd) {
 
-				cmd.Spec = "[-f]"
+				cmd.Spec = "[-f] [--chart]"
```

Comment:
should the --chart option jus be on `inspect` instead of each subcommand? same goes for -f, really.

Reply:
--charts is also useful for apply and template though right? maybe I'm misunderstanding...

Reply:
oh yeah I read this wrong. I thought this was in 3 places because it was in each of the 3 subcommands of `inspect`, but it's actually in `template, apply, inspect`. all is well.

Reply:
Yeah, I had been avoiding that because I thought it'd be easier from a workflow perspective to be able to add all flags at the end of the command. Conceptually it does make more sense to attach it to the parent command, and it makes for cleaner code, so will change.